test/views/declaration/LocationViewSpec.scala

```diff
@@ -401,4 +401,39 @@ class LocationViewSpec extends UnitViewSpec with ExportsTestData with Stubs with
       view.getElementById("city").attr("value") must be(lcity)
     }
   }
+
+  "Goods Location view" should {
+    def viewWithCorrectBackButton(declarationType: DeclarationType, redirect: Call): Unit =
+      "have correct back-link" when {
+        val view = createView(declarationType = declarationType)
+        "display 'Back' button that links to correct page" in {
+          val backButton = view.getElementById("back-link")
+
+          backButton must containText("site.back")
+          backButton.getElementById("back-link") must haveHref(redirect)
+        }
+      }
+
+    "for Standard declaration" should {
+      behave like viewWithCorrectBackButton(DeclarationType.STANDARD, controllers.declaration.routes.RoutingCountriesSummaryController.displayPage())
```

Comment:
`behave like` thats new

What's that do?

Reply:
It's not that new - it's used in a couple of places.   Basically used to create parameterised blocks of code that can be re-used